include/OpenColorIO/OpenColorIO.h

```diff
@@ -1123,6 +1123,12 @@ extern OCIOEXPORT std::ostream& operator<< (std::ostream&, const Config&);
 class FileRules
 {
 public:
+
+    /// Reserved rule name for the default rule.
+    static const char * DefaultRuleName;
+    /// Reserved rule name for the file path search rule \see FileRules::insertPathSearchRule.
+    static const char * FilePathSearchRuleName;
```

Comment:
Wondering if these should go in the `extern OCIOEXPORT` section of `OpenColorTypes.h` instead for consistency with other static strings. Just a question for discussion - don't feel strongly about it.

Reply:
Good point. I thought (and still think) that this is the right location as it clearly demonstrates it can only be used by `FileRules` rule name. And that cannot be explicitly changed, modified, etc.. It's only helpers when looping through the rule names to easily detect these two 'special' rules.